e2e/config.yaml

```diff
@@ -9,3 +9,5 @@ pvcspecs:
   - name: cephfs
     storageclassname: rook-cephfs
     accessmodes: ReadWriteMany
+    unsupportedDeployers:
```

Comment:
I prefer limiting the config file to user preferences. Whether a storageClass is not supported by a deployer is a function of the code not config.

Reply:
So in combination with my other comment, the helper function can decide in code if a pvcspec name is incompatible with a workload,deployer config.

Reply:
The code does not have any way to tell if something is supported or not. This is pure configuration.

When this is set in the configuration, I can create multiple configurations. For If I work on rbd related stuff, I can run the test only with rbd, without using cryptic -run arguments. This is much more powerful that doing this in code.